eden/scm/edenscm/ext/github/submit.py

```diff
@@ -262,11 +262,13 @@ async def rewrite_pull_request_body(
 
     head_commit_data = partition[0]
     commit_msg = head_commit_data.get_msg()
+
     title, body = create_pull_request_title_and_body(
         commit_msg,
         pr_numbers_and_num_commits,
         index,
         repository,
+        template=ui.config("pr", "template", "")
```

Comment:
I actually can't figure out how to set this config option, I keep hitting the `if not template` code path. Would love a some pointers on how to pass a custom template so I can prove this works :)